server/channels/store/sqlstore/link_metadata_store.go

```diff
@@ -16,10 +16,26 @@ import (
 
 type SqlLinkMetadataStore struct {
 	*SqlStore
+
+	linkMetadataQuery sq.SelectBuilder
 }
 
 func newSqlLinkMetadataStore(sqlStore *SqlStore) store.LinkMetadataStore {
-	return &SqlLinkMetadataStore{sqlStore}
+	s := &SqlLinkMetadataStore{
+		SqlStore: sqlStore,
+	}
+
+	s.linkMetadataQuery = s.getQueryBuilder().
+		Select(
+			"Hash",
+			"URL",
+			"Timestamp",
+			"Type",
+			"Data",
+		).
```

Comment:
Not blocking, but having [recently](https://github.com/mattermost/mattermost-plugin-calls/pull/990/files) gone through similar changes for Calls, I've found it helpful to encode the expected columns once and re-use the slice in both insert and select code paths rather than duplicating.

Reply:
Great idea! Thank you :)